io.github.kriptolix.Poliedros.json

```diff
@@ -0,0 +1,39 @@
+{
+    "id" : "io.github.kriptolix.Poliedros",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "47",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "poliedros",
+    "finish-args" : [
+        "--share=network",
```

Comment:
why does this need network access?

Reply:
No need, I just forgot to remove. I'm going to remove.